webapp/packages/plugin-authentication-administration/src/Administration/Users/UserForm/Info/UserFormInfoPart.ts

```diff
@@ -132,6 +138,8 @@ export class UserFormInfoPart extends FormPart<IUserFormInfoState, IUserFormStat
   }
 
   private async updateCredentials() {
+    this.state.password = this.state.password.trim();
```

Comment:
please do not modify state object
`const password = this.state.password.trim();`